compiledb-gen-parser

```diff
@@ -103,6 +106,10 @@ def parse_flags(build_log, proj_dir, inc_prefix, verbose):
     dir_stack = [proj_dir]
     working_dir = proj_dir
 
+    exclude_regex_list = []
+    for item in itertools.chain.from_iterable(exclude_list):
+        exclude_regex_list.append(re.compile(item))
+
```

Comment:
I think we can simplify this fix. Can you please use the same approach as in:

https://github.com/nickdiego/compiledb-generator/blob/47f5a06338a8f97409ca52d0a676dad2a192df7b/compiledb-gen-parser#L82-L91

And then let regex module to do the trick (e.g: `exclude_list_regex.match(...)`) to decide if a file should be added or not to compilation db.